source/code/include/playfab/PlayFabPlatformUtils.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <chrono>
+#include <ctime>
+#include <iomanip>
+#include <sstream>
+
+namespace PlayFab
+{
+    typedef std::chrono::time_point<std::chrono::system_clock> TimePoint;
+
+    inline TimePoint GetPlayFabTimePointNow()
+    {
+        return std::chrono::system_clock::now();
+    }
+
+    inline time_t GetPlayFabTimeTNow()
+    {
+        time_t now = std::chrono::system_clock::to_time_t(GetPlayFabTimePointNow());
+#if defined(PLAYFAB_PLATFORM_IOS) || defined(PLAYFAB_PLATFORM_ANDROID) || defined(PLAYFAB_PLATFORM_LINUX) || defined(PLAYFAB_PLATFORM_SWITCH)
+        return static_cast<Json::Int64>(now);
```

Comment:
>static_cast<Json::Int64> [](start = 15, length = 24)

the return type is time_t which `now` is already typed as. what is this static_cast for?

Reply:
No idea. I haven't tested it on those platforms yet.
I'm just copy pasting the code that was already there.